blueprints/-utils.js

```diff
@@ -1,9 +1,33 @@
 const { dasherize } = require('ember-cli-string-utils');
+const { EOL } = require('os');
+
+function generateComponentSignature(componentName) {
+  let args = `  // The arguments accepted by the component${EOL}  Args: {};`;
+
+  let blocks =
+    `  // Any blocks yielded by the component${EOL}` +
+    `  Blocks: {${EOL}` +
+    `    default: []${EOL}` +
+    `  };`;
+
+  let element =
+    `  // The element to which \`...attributes\` is applied in the component template${EOL}` +
+    `  Element: null;`;
+
+  return (
+    `interface ${componentName}Signature {${EOL}` +
```

Comment:
Generally the guidance we've given with Glint is to always export your signature, since it's useful when you start talking about places where other components are yielding or or accepting this one as an arg.

Reply:
Thanks @dfreeman. Updated accordingly.